### ISSUE TITLE
Feature Request: Collapsible Admonition blocks.

### ISSUE THREAD

**Issue by TomPeirs** [elviswolcott/remark-admonitions#44]
It would be interesting to enable admonitions to be collapsible.

Perhaps we could draw inspiration from the following:
https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks

Instead of defining 3 colons `:::` we could define 3 question marks `???` and then they are collapsible (collapsed by default)
Only the title would be shown, and an arrow indicator on the right of the admonition shows the user that it is possible to expand the admonition.

**Comment by homotechsual** [comment]
Yes please - this would really assist me in migrating documentation sites from `mkdocs-material` to Docusaurus!
